src/extension/background-script/WelcomeService.ts

```diff
@@ -89,19 +95,40 @@ export async function createNewIdentityByMnemonicWord(whoAmI: ProfileIdentifier,
 }
 
 /**
- *
  * Recover new identity by a password and mnemonic words
  *
- * @param whoAmI Who Am I
  * @param password password used to generate mnemonic word, can be empty string
  * @param word mnemonic words
+ * @param info additional information
  */
 export async function restoreNewIdentityWithMnemonicWord(
-    whoAmI: ProfileIdentifier,
     word: string,
     password: string,
-): Promise<void> {
-    await generateNewIdentity(whoAmI, await recover_ECDH_256k1_KeyPair_ByMnemonicWord(word, password))
+    info: {
+        whoAmI?: ProfileIdentifier
+        nickname?: string
+        localKey?: JsonWebKey
+        details?: LinkedProfileDetails
+    },
+): Promise<PersonaIdentifier> {
+    const key = await recover_ECDH_256k1_KeyPair_ByMnemonicWord(word, password)
+    const pubJwk = await CryptoKeyToJsonWebKey(key.key.publicKey)
```

Comment:
Can `pubJwk `, `privJwk ` and `localKeyJwk` working in parallel?

Reply:
@Jack-Works

Reply:
Yes, but unnecessary. First, `CryptoKeyToJsonWebKey` and `JsonWebKeyToCryptoKey` have cache. Secondly, those operations are fast. For example, import and export only takes about 1ms and 0.8ms on my computer. So I think it is acceptable to make the code more clear instead of use `Promise.all`